modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SkuResourceImpl.java

```diff
@@ -344,7 +344,7 @@ private Sku _updateSKU(CPInstance cpInstance, Sku sku) throws Exception {
 		DateConfig expirationDateConfig = new DateConfig(expirationCalendar);
 
 		cpInstance = _cpInstanceService.updateCPInstance(
-			cpInstance.getExternalReferenceCode(), cpInstance.getCPInstanceId(),
+			sku.getExternalReferenceCode(), cpInstance.getCPInstanceId(),
```

Comment:
@victorcdp , Please resend me a PR to account for the following scenarios.

1. if ERC is provided (with value) the ERC of the SKU is updated with the new value
2. if ERC is not provided in the JSON the ERC of the cpInstance is not updated.
3. if ERC is provided as an empty string, the ERC of the cpInstance is not updated.